error-issue/index.ts

```diff
@@ -19,19 +19,29 @@ require('dotenv').config();
 import express from 'express';
 import statusCodes from 'http-status-codes';
 import {ErrorIssueBot} from './src/bot';
+import {StackdriverApi} from './src/stackdriver_api';
+import {ErrorMonitor, ERROR_ISSUE_ENDPOINT} from './src/error_monitor';
 import {ErrorReport} from './src/types';
 
 const GITHUB_REPO = process.env.GITHUB_REPO || 'ampproject/amphtml';
 const [GITHUB_REPO_OWNER, GITHUB_REPO_NAME] = GITHUB_REPO.split('/');
 const GITHUB_ACCESS_TOKEN = process.env.GITHUB_ACCESS_TOKEN;
+const PROJECT_ID = process.env.PROJECT_ID || 'amp-error-reporting';
 
 const bot = new ErrorIssueBot(
   GITHUB_ACCESS_TOKEN,
   GITHUB_REPO_OWNER,
   GITHUB_REPO_NAME
 );
 
-module.exports.app = async (req: express.Request, res: express.Response) => {
+const stackdriver = new StackdriverApi(PROJECT_ID);
+const monitor = new ErrorMonitor(stackdriver);
+
+/** Endpoint to create a GitHub issue for an error report. */
+module.exports.errorIssue = async (
```

Comment:
style nit: combine all exports into `module.exports = { }` at the end of the file; makes it easier to see what's being exported

Reply:
Switched to `export async function` per the standard across the repo